mmdet/models/detectors/cascade_rcnn.py

```diff
@@ -270,7 +298,26 @@ def forward_train(self,
         return losses
 
     def simple_test(self, img, img_meta, proposals=None, rescale=False):
+        """Run inference on a single image.
+
+        Args:
+            img (Tensor): must be in shape (B, C, H, W)
+            img_meta (list[dict]): a list with one dictionary element.
+                See `mmdet/datasets/pipelines/formatting.py:Collect` for
+                details of meta dicts.
+            proposals : if specified overrides rpn proposals
+            rescale (bool): if True returns boxes in original image space
+
+        Returns:
+            dict: results
+        """
         x = self.extract_feat(img)
+
+        if isinstance(img_meta, dict):
+            # a single dict was passed in but the rest of this function
+            # expect a list of with one item.
+            img_meta = [img_meta]
```

Comment:
`simple_test` is assumed to be called from `forward_test` and `img_meta` is supposed to be a list of dict.